requirements.txt

```diff
@@ -1,3 +0,0 @@
-Django>=1.8
```

Comment:
I did not see this change when I first reviewed the pull request. Even though the requirements file is no longer needed by tox, it is useful for developers who want to develop/hack this project. Is there another reason why you removed requirements.txt?

Reply:
I've moved them into the tox configuration so I can leverage [this feature](http://tox.readthedocs.io/en/latest/example/basic.html#compressing-dependency-matrix) to run tests for the following combinations:

- Python 2.7 with Django 1.8
- Python 2.7 with Django 1.11
- Python 3.5 with Django 1.8
- Python 3.5 with Django 1.11
- Python 3.6 with Django 1.8
- Python 3.6 with Django 1.11

This change should not affect `python setup.py install` and `python setup.py develop`, since they don't seem to use the requirements file in any way.

If I understand you correctly, you'd still like to be able to run `pip install -r requirements.txt` as a standalone command in your dev environment. If you know of a convenient way to have tox load the requirements from the file while still testing against multiple Django versions, I could bring it back. I'd like to avoid duplicating the requirements though.

Reply:
> If I understand you correctly, you'd still like to be able to run pip install -r requirements.txt as a standalone command in your dev environment.

Correct.

I don't have a string opinion against repeating the requirements considering that they are used by different tools, but in the past I've written code in setup.py that extracts the dependencies from requirements.txt. The downside is that requirements.txt must now be included in the source and wheel dists. Should we keep this as a separate issue?

Reply:
> Should we keep this as a separate issue?

Meaning you'd like to keep the changes to tox.ini but prefer leaving the requirements file in place for now?

Reply:
Yes exactly: the changes to tox.ini are great, but until we have a robust solution we do not want to remove requirements.txt